backend/tracim_backend/tests/functional/test_elasticsearch_search.py

```diff
@@ -1185,7 +1183,7 @@ def test_api__elasticsearch_workspace_search__ok__check_result(
             (("bob", "password"), {"search_string": "bob", "page_nb": 2, "size": 1}, [2], 2),
             (("bob", "password"), {"search_string": "bob", "size": 0}, [], 2),
             (("bob", "password"), {"search_string": "bob riyad", "owner_ids": [3]}, [], 0),
-            (("bob", "password"), {"search_string": "bob riyad", "owner_ids": [2]}, [1, 2], 2),
+            (("bob", "password"), {"search_string": "bob riyad", "owner_ids": [2]}, [2, 1], 2),
```

Comment:
Can you use sets if the order is not what you are testing?

assert set(...) == set(..)

Reply:
Order here should not change if order do change here it's a bug. Don't understood why this has not been detected before.